nextcloud_install_production.sh

```diff
@@ -2,6 +2,74 @@
 
 # T&M Hansson IT AB © - 2020, https://www.hanssonit.se/
 
+# shellcheck disable=2015,2034,2059
+true
+
+# Variables:
+# Todo: make DOWNLOAD_DIR readable again; is disabled for now to fix travis
+SCRIPTS=/var/scripts
+#DOWNLOAD_DIR=/tmp/NC_VM
+
+# Download and validate Nextcloud VM
+if ! [ -d $SCRIPTS ]
+then
+    # Check if root
+    if [[ "$EUID" -ne 0 ]]
+    then
+        echo "You have to run this script as root! Exiting..."
+        exit 1
+    fi
+
+    # Test internet connection
+    if ! ping -c1 -W1 github.com &>/dev/null
+    then
+        echo "Couldn't reach github.com. Exiting..."
+        exit 1
+    fi
+
+    # Download the release package (.tar.gz) file
+    # Todo: Remove the hashtags after it is released and change the download path if necessary
+    #rm -rf $DOWNLOAD_DIR
+    #mkdir $DOWNLOAD_DIR
+    #cd $DOWNLOAD_DIR
+    #curl -sLO https://github.com$(curl -s -m 900 https://github.com/nextcloud/vm/tags/ | grep -om1 "/nextcloud/vm/archive/[0-9.]*.tar.gz")
+
+    # Todo: Verify the state of the downloaded package with a checksum
+
+    # Todo: Verify the release package and integrity with the gpg key?
```

Comment:
I guess what #11 talks about would then be handled here. So currently, there is not much for me to review.